docs/cosmology/io/index.rst

```diff
@@ -0,0 +1,156 @@
+.. _cosmology_io:
+
+*********************
+Cosmology I/O/Convert
```

Comment:
This looks a little odd. Was it supposed to be something like `Cosmology I/O and Convert`?

Reply:
I could go either way. I/O is common parlance, but I want to distinguish reading and writing from live-object conversion, like the Cosmology -> Mapping so I added `/Convert` to the standard abbreviation.

Reply:
Changed!